goldilocks_getter.py

```diff
@@ -144,16 +171,25 @@ def get_html(
         Bytes from Goldilocks.
     """
 
-    # This may need to be adjusted depending on your naming scheme.
-    cluster_name = "-".join(cluster.split("-")[:-1])
-    try:
-        endpoint = f"https://goldilocks-{cluster_name}.{domain}/dashboard/{namespace}"
-        request = requests.get(endpoint)
-    except requests.exceptions.ConnectionError:
-        print(f"Exception raised when connecting to {endpoint}")
-        raise SystemExit
+    # Test multiple hostname from domain separators.
+    # The endpoint should normally be a FQDN (e.g. use a dot).
+    separators = ["-", "."]
 
-    return request.content
+    for index, sep in enumerate(separators):
+        try:
+            endpoint = f"https://goldilocks.{cluster}{sep}{domain}/dashboard/{namespace}"
```

Comment:
Again, specific use case for me, but our endpoints are set up as `goldilocks-{cluster}...`, not as a subdomain. Thus, this breaks. Additionally, in my kubeconfig, they're labeled as `{env}-{region}-{color}`, while the endpoint omits the color. Thus, if I pass in `test-us-west-2-green` as the cluster, this attempts to connect to the color. Something like this would fix that:

`if cluster.endswith(("blue", "green")):
    cluster = "-".join(cluster.split("-")[:-1])
`